salt/modules/inspectlib/collector.py

```diff
@@ -153,7 +156,7 @@ def _get_managed_files(self):
         links = set()
         files = set()
 
-        for line in os.popen("rpm -qlav").xreadlines():
+        for line in os.popen("rpm -qlav"):
```

Comment:
I was actually wondering that too. I meant to look further into it, but seems I forgot.

@isbm Can you comment about using `os.popen()` here? Should that be `salt.utils.popen` instead?